src/components/OwnerPanel/OwnerPanel.tsx

```diff
@@ -45,6 +53,9 @@ const OwnerPanel = (props: Props) => {
     return props.extraInfo[current] != null;
   });
 
+  const location = window.location.href;
+  const facebookQuote = 'Help raise money for ' + props.sellerName;
```

Comment:
I just improvised this on the spot...any suggestions?

Reply:
gonna slack you the icons we've been using

Reply:
Chinatown has been suffering from the effects of COVID. Help keep Chinatown alive by donating/buying a gift card! Check out <Insert Merchant Name>'s page:

idk something like that?